docs/serverless/AI-for-security/llm-performance-matrix.mdx

```diff
@@ -8,11 +8,11 @@ status: in review
 
 This table describes the performance of various large language models (LLMs) for different use cases in ((elastic-sec)), based on our internal testing. To learn more about these use cases, refer to <DocLink slug="/serverless/security/attack-discovery" text="Attack discovery"/> or <DocLink slug="/serverless/security/ai-assistant" text="AI Assistant"/>.
 
-|           **Feature**        | **Model**             |                    |                    |            |                 |
-|-------------------------------|-----------------------|--------------------|--------------------|------------|-----------------|
-|                              | **Claude 3: Opus** | **Claude 3.5: Sonnet** | **Claude 3: Haiku** | **GPT-4o** | **GPT-4 Turbo** |
-| **Assistant: general**       | Excellent              | Excellent          | Excellent          | Excellent  | Excellent       | 
-| **Assistant: ((esql)) generation** | Great            | Great              | Poor               | Excellent  | Poor            | 
-| **Assistant: alert questions** | Excellent            | Excellent          | Excellent          | Excellent  | Poor            |
-| **Attack discovery**          | Excellent             | Excellent          | Poor               | Poor       | Good            | 
+|           **Feature**        | **Model**             |                    |                    |            |                 |                   |           |  
+|-------------------------------|-----------------------|--------------------|--------------------|------------|-----------------|------------------|-----------|
+|                              | **Claude 3: Opus** | **Claude 3.5: Sonnet** | **Claude 3: Haiku** | **GPT-4o** | **GPT-4 Turbo** | Gemini 1.5 Pro | Gemini 1.5 Flash |
+| **Assistant: general**       | Excellent              | Excellent          | Excellent          | Excellent  | Excellent       | Excellent        | Excellent |
+| **Assistant: ((esql)) generation** | Great            | Great              | Poor               | Excellent  | Poor            | Good              | Poor    |
+| **Assistant: alert questions** | Excellent            | Excellent          | Excellent          | Excellent  | Poor            | Excellent         | Good    |
+| **Attack discovery**          | Excellent             | Excellent          | Poor               | Poor       | Good            | Great             | Poor     |
```

Comment:
I might have said this before, but considering how tricky tables can be, this is a great, clean presentation of info!